server/text_generation_server/models/custom_modeling/flash_llama_modeling.py

```diff
@@ -261,9 +277,8 @@ def forward(
         )
         query = query.view(-1, self.num_heads, self.head_size)
         kv = kv.view(-1, 2, self.num_key_value_heads, self.head_size)
-
-        self.rotary_emb(query, cos, sin)
-        self.rotary_emb(torch.select(kv, dim=1, index=0), cos, sin)
```

Comment:
Note that this piece is changed to accomodate both flash-attn rotary and vllm rotary APIs. Hopefully this change is fine.